post/2023ironman-30/index.md

```
@@ -16,19 +16,19 @@ date: 2023-10-14
 
 在接下來的十幾天，我們製作了許多令人驚豔的效果。每一天專案的背後都會放入一些實用的思維，語法的應用，以及一些小技巧。希望用效果的視覺包裝可以讓大家更容易理解與覺得有趣，並且能夠在未來的開發中運用。
 
-![](final.gif)
+![](../2023ironman-18/final.gif)
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)